### DISCUSSION TITLE
Quais são os passos para criação de um projeto como este?

### DISCUSSION THREAD

**Discussion by davidborghi** [andreroggeri/pynubank#396] [Q&A] [answered]
Olá, espero que estejam bem! 

Não sou programador, por isso a dúvida.

Sempre fico pensando quais são os passos necessários para criar um projeto como este, considerando que o Nubank não dispõe de documentação oficial, ainda que oficialmente tenha reconhecido que usa API's públicas. 

Não sei se ficou entendível, mas refiro-me aos argumentos, funções etc.

Caso tenha algum material de leitura que auxilie, se possível, por favor, fique à vontade para recomendar.

Obrigado pela atenção e disposição!

Desculpe-me pelo incômodo!

Tudo de bom!

**Comment by andreroggeri** [comment, marked as answer]
Muito tempo livre hehehehe

Eu comecei fazendo a engenharia reversa do site do Nubank. Nesse caso é bem simples, bastando usar o próprio navegador pra ver as requisições que são enviadas e codifiquei em python. 
Pro app é um pouco mais complicado pq tem alguns mecanismos de segurança pra impedir a inspeção de pacotes, mas eu dei uma detalhada [nessa discussion](https://github.com/andreroggeri/pynubank/discussions/325) como desativar eles.

**Comment by davidborghi** [comment]
Obrigado pela resposta! 

Eu achava que era por engenharia reversa mesmo, mas achei que já havia uma possibilidade de fazer algo menos "massante", mas imagino que use alguma outra API para organizar as requisições, ou não?

Obrigado pelas informações, atenção e disposição! 

Desculpe-me pelo incômodo!

Parabéns pelo trabalho!
